### PR TITLE
Enhancements and fixes to documentation

### PR DESCRIPTION
**Pull request by ljessicc**
Please preview the changed/updated page/image links, and other miscellaneous changes to the doc/ files

### CODE REVIEW

File: md-docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -46,9 +46,9 @@ By adding this simple comment, you are telling the community that you wrote the
...
 
-We don't place explicit copyright statements in the project source files.  The project comprises many distinct pieces of code, spread across numerous source files, and authored by a variety of individuals.  Managing copyright statements is unproductive and [can lead to confusion and contention around the edge cases](https://opensource.com/law/14/n2/copyright-statements-source-files).  Rather we utilize [the NOTICE file](https://github.com/AdoptOpenJDK/openjdk-build/blob/master/NOTICE) mechanism as a way to acknowledge copyright broadly where there is a valid reason to do so.
+We don't place explicit copyright statements in the project source files.  The project comprises many distinct pieces of code, spread across numerous source files, and authored by a variety of individuals.  Managing copyright statements is unproductive and can lead to confusion and contention around the edge cases.  Rather we utilize [the NOTICE file](https://github.com/AdoptOpenJDK/openjdk-build/blob/master/NOTICE) mechanism as a way to acknowledge copyright broadly where there is a valid reason to do so.
```
Comment (karianna) [inline md-docs/CONTRIBUTING.md:49]: Not sure why the first link was removed and the second one changed?

File: md-docs/FAQ.md [hunk 2]
```diff
@@ -9,20 +9,20 @@ repository.
 The GitHub teams relevant to this repository are as follows (Note, you
 won't necessarily have access to see these links):
 
-- [GetOpenJDK](https://github.com/orgs/AdoptOpenJDK/teams/getopenjdk) - `Triage` level of access which lets you assign issues to people
-- [build](https://github.com/orgs/AdoptOpenJDK/teams/build) - `Write` access which lets you approve and merge PRs and run and configure most Jenkins jobs
-- [release](https://github.com/orgs/AdoptOpenJDK/teams/build) - Allows you to run the release jobs in Jenkins
+- GetOpenJDK- `Triage` level of access which lets you assign issues to people
```
Comment (karianna) [inline md-docs/FAQ.md:12]: There are actually named teams for this repo.  Suggest looking at the AdoptOpenJDK/Teams tab in GitHub and see if you can transpose that info here.

File: md-docs/README.md [hunk 3]
```diff
@@ -9,17 +9,18 @@ AdoptOpenJDK makes use of these scripts to build binaries on the build farm at
 ## TL;DR: I want to build a JDK NOW!
 
 ### Build jdk natively on your system
-
-To do this you will need to have your machine set up with a suitable
-compiler and various other tools available. We set up our machines using
+Have your machine set up with a suitable
+compiler and other tools available.
+We set up our machines using:
```
Comment (karianna) [inline md-docs/README.md:14]: The below is not a list so the `:` is not needed.

File: md-docs/README.md [hunk 4]
```diff
@@ -9,17 +9,18 @@ AdoptOpenJDK makes use of these scripts to build binaries on the build farm at
 ## TL;DR: I want to build a JDK NOW!
 
 ### Build jdk natively on your system
-
-To do this you will need to have your machine set up with a suitable
-compiler and various other tools available. We set up our machines using
+Have your machine set up with a suitable
+compiler and other tools available.
```
Comment (karianna) [inline md-docs/README.md:13]: Can haul this line up

File: md-docs/README.md [hunk 5]
```diff
@@ -9,17 +9,18 @@ AdoptOpenJDK makes use of these scripts to build binaries on the build farm at
...
 
-Once you've got all of the prerequisites installed, clone this openjdk-build
-repository (`git clone https://github.com/AdoptOpenJDK/openjdk-build` and
-kick off a build a follows with this script. The `-J` parameter specifies
-the "boot JDK" which should generally be one major version prior to the one
-you are building (although one of the same major version will also work).
+Once all of the prerequisites are installed, clone this openjdk-build
+repository (`git clone https://github.com/AdoptOpenJDK/openjdk-build`) 
+
+kick off a 'build a follows' with this script.
```
Comment (karianna) [inline md-docs/README.md:21]: Suggest capitilizing kick.  Not sure that `build a follows` makes sense here.

File: md-docs/README.md [hunk 6]
```diff
@@ -34,9 +35,9 @@ e.g.
 
 ## How do I build OpenJDK in a docker image?
 
-If you do not want to set up your machine with all the prerequisites for
-building OpenJDK, you can use our docker images under the [docker]
-directory as follows (first version builds HotSpot, second builds J9 - the
+If unable to set up your machine with all the prerequisites for
```
Comment (karianna) [inline md-docs/README.md:38]: or unwilling to pollute your host machine environment

File: md-docs/README.md [hunk 7]
```diff
@@ -45,42 +46,37 @@ as we can generate valid dockerfile for it):
...
 ## Repository contents
 
-This repository contains several useful scripts in order to build OpenJDK
+This repository contains useful scripts used to build OpenJDK
 personally or at build farm scale.
 
 1. The `build-farm` folder contains shell scripts for multi configuration Jenkins
 build jobs used for building Adopt OpenJDK binaries.
-1. The `docker` folder contains tools for generating dockerfiles which can be used as part of building
+2. The `docker` folder contains tools for generating dockerfiles which can be used as part of building
```
Comment (karianna) [inline md-docs/README.md:56]: You can leave the number as 1. - Markdown knows how to convert it

File: md-docs/README.md [hunk 8]
```diff
@@ -120,7 +116,7 @@ For reference, OpenJDK version numbers look like 1.8.0_162-b12 (for Java 8) or
 
 --build-variant <variant_name>
 specify a OpenJDK build variant, e.g. openj9.
-For reference, the default variant is hotspot and does not need to be specified.
+For reference, the default variant is hotspot(needs no specification).
```
Comment (karianna) [inline md-docs/README.md:119]: I think the previous sentence is clearer

**Review by karianna — CHANGES_REQUESTED** [review]
Didn't cover all of it, but a little more fine-tuning required :-)

**Comment by karianna** [comment]
Also, can you please edit this title with a description of the change :-)

File: md-docs/FAQ.md [hunk 9]
```diff
@@ -41,7 +41,8 @@ There is also some documentation in [CHANGELOG.md](CHANGELOG.md)
 
 ## What are the prerequisites for a system used for builds?
 
-- The upstream OpenJDK build requirements are at [Supported Build Platforms](https://wiki.openjdk.java.net/display/Build/Supported+Build+Platforms)
+- The upstream OpenJDK build requirements are at [Supported Build Platforms](https://wiki.open
+jdk.java.net/display/Build/Supported+Build+Platforms)
```
Comment (M-Davies) [inline md-docs/FAQ.md:45]: The markdown link does not take across multilines

File: md-docs/RELEASING.md [hunk 10]
```diff
@@ -49,7 +49,6 @@ Don't be scared off by this document! If you already understand the stuff in th
      - Announce: https://mail.openjdk.java.net/pipermail/jdk8u-dev/
    - jdk11u:  https://hg.openjdk.java.net/jdk-updates/jdk11u/
      - Announce: https://mail.openjdk.java.net/pipermail/jdk-updates-dev/
-   - jdkNN: https://hg.openjdk.java.net/jdk/jdkNN/
      - Announce: https://mail.openjdk.java.net/pipermail/jdk-dev/
```
Comment (M-Davies) [inline md-docs/RELEASING.md:52]: This can probably be kept, the NN refers to whatever the next JDK number

File: md-docs/RELEASING.md [hunk 11]
```diff
@@ -220,26 +219,26 @@ The following examples all use `-m1` as an example - this gets replaced with a l
...
   
-  1. Build the `pipeline_jobs_generator` that you just made. Ensure the equivalent `openjdkxx_pipeline` to the generator exists or this will fail. If the job fails or is unstable, search the console log for `WARNING` or `ERROR` messages for why. Once it has completed successfully, the [pipeline](https://ci.adoptopenjdk.net/job/build-scripts/) is ready to go!
+  4. Build the `pipeline_jobs_generator` that you just made. Ensure the equivalent `openjdkxx_pipeline` to the generator exists or this will fail. If the job fails or is unstable, search the console log for `WARNING` or `ERROR` messages for why. Once it has completed successfully, the [pipeline](https://ci.adoptopenjdk.net/job/build-scripts/) is ready to go!
 
-  1. Update the view for the [build and test pipeline calendar](https://ci.adoptopenjdk.net/view/Build%20and%20Test%20Pipeline%20Calendar) to include the new version.
+  5. Update the view for the [build and test pipeline calendar](https://ci.adoptopenjdk.net/view/Build%20and%20Test%20Pipeline%20Calendar) to include the new version.
```
Comment (M-Davies) [inline md-docs/RELEASING.md:233]: Same thing pointed out by @karianna

**Review by M-Davies — CHANGES_REQUESTED** [review]
Great start. Got a few points of my own.
- Could probably just called the md-docs folder `docs/` as it's generally more obvious that the folder contains docs that way
- See below

**Comment by ljessicc** [comment]
Alright I will work on these. Thanks @karianna

File: md-docs/CONTRIBUTING.md (hunk 1 above)
Comment (ljessicc) [reply md-docs/CONTRIBUTING.md:49]: When I tried accessing the link, it stated that the page no longer exists and I could not find an appropriate link to replace it

File: md-docs/FAQ.md (hunk 2 above)
Comment (ljessicc) [reply md-docs/FAQ.md:12]: I can't see the teams tab. 
image

Comment (ljessicc) [reply md-docs/FAQ.md:12]: I don't see the teams tab. 
image

Comment (ljessicc) [reply md-docs/FAQ.md:12]: could it be because I'm not part of any team?

**Comment by ljessicc** [comment]
could please review the updated changes as per previous reviews? Renamed md-docs to docs/, replaced and updated deleted links

**Review by M-Davies — CHANGES_REQUESTED** [review]
One final comment (sorry I missed this in my original review) and then LGTM

File: docs/FAQ.md [hunk 12]
```diff
@@ -9,20 +9,20 @@ repository.
 The GitHub teams relevant to this repository are as follows (Note, you
 won't necessarily have access to see these links):
 
-- [GetOpenJDK](https://github.com/orgs/AdoptOpenJDK/teams/getopenjdk) - `Triage` level of access which lets you assign issues to people
-- [build](https://github.com/orgs/AdoptOpenJDK/teams/build) - `Write` access which lets you approve and merge PRs and run and configure most Jenkins jobs
-- [release](https://github.com/orgs/AdoptOpenJDK/teams/build) - Allows you to run the release jobs in Jenkins
+-[GetOpenJDK](https://github.com/orgs/AdoptOpenJDK/teams/getopenjdk) `Triage` level of access which lets you assign issues to people
+-[build](https://github.com/orgs/AdoptOpenJDK/teams/build) `Write` access which lets you approve and merge PRs and run and configure most Jenkins jobs
+-[release](https://github.com/orgs/AdoptOpenJDK/teams/build)- Allows you to run the release jobs in Jenkins
```
[the remark this answers is not in the document]
Comment (ljessicc) [reply docs/FAQ.md:14]: Ok got it. Thank you @M-Davies.

File: docs/CHANGELOG.md [hunk 13]
```diff
@@ -6,7 +6,7 @@
 
 ## Version 1.0.0 (14th May 2018)
 
-See [Commit History](https://github.com/AdoptOpenJDK/openjdk-build/commits/master) 
+See [Commit History](https://github.com/AdoptOpenJDK/open jdk-build/commits/master)
```
Comment (M-Davies) [inline docs/CHANGELOG.md:9]: Did you mean to add this space? It makes the link invalid is all

**Review by M-Davies — CHANGES_REQUESTED** [review]

**Review by M-Davies — APPROVED** [review]

**Comment by ljessicc** [comment]
I did my final commits after correcting generated lint code errors. If there's anything else is required, I'd be glad to.

**Comment by ljessicc** [comment]
please review and merge changes if ok. Thank you

**Comment by karianna** [comment]
You have a merge conflict

**Comment by ljessicc** [comment]
@karianna ok, looking into it. Thank you

**Comment by karianna** [comment]
@ljessicc looks like merge conflicts still sorry :-(

**Review by sxa — COMMENTED** [review]
I'm not convinced by the moving of the `.md` files at the top level. I think `README.md` `CONTRIBUTING.md` and `FAQ.md` should definitely stay at the top level of the project.

**Comment by karianna** [comment]
This has been superseded - thanks for your submission!
